src/jit/compiler.cpp

```diff
@@ -5790,8 +5790,9 @@ int Compiler::compCompileHelper(CORINFO_MODULE_HANDLE            classPtr,
     }
     info.compRetNativeType = info.compRetType = JITtype2varType(methodInfo->args.retType);
 
-    info.compCallUnmanaged   = 0;
-    info.compLvFrameListRoot = BAD_VAR_NUM;
+    info.compUnmanagedCallCount            = 0;
```

Comment:
Do we need to count both `compUnmanagedCallCount` and `compSuppressGCTransitionCallCount`? I have done a cursory review of where this is used, and it seems like all places care about the number of calls with GC transitions only.

Reply:
But that isn't all cases that may be interesting here. We have the case where a function contains one function that doesn't want the transition and one that does. In that case the call doesn't need the transition, but it still needs to the frame. See lower.cpp in `DoPhase()` and `LowerNonvirtPinvokeCall()`

Reply:
> a function contains one function that doesn't want the transition and one that does

In this case, we would have one call with GC transition and it would make `DoPhase()` and friends emit a frame. I am not following why it does not work.

`Do you agree with the simplification I have suggested for `compMethodRequiresPInvokeFrame` above?

Reply:
I do agree with the suggestion. The nuance was not related to right now but in future scenarios where interest in the delta is desired. For right now, I will combine the two values into one - `compUnmanagedCallCountWithGCTransition`.